ClavaWeaver/test/pt/up/fe/specs/cxxweaver/tests/CApiTest.java

```diff
@@ -143,11 +143,6 @@ public void testStaticCallGraph() {
         newTester().test("StaticCallGraphTest.js", "static_call_graph.c");
     }
 
-    @Test
-    public void testPassComposition() {
-        newTester().test("PassCompositionTest.js", "pass_composition.c");
-    }
-
```

Comment:
Keep, this is an important test

Reply:
Re-added in 4612a1f3b and refactored to remove erroneous code in ef6db7680.